src/platform/packages/shared/response-ops/rule_form/src/translations.ts

```diff
@@ -110,6 +110,14 @@ export const ALERT_LINK_DASHBOARDS_TITLE = i18n.translate(
   }
 );
 
+export const ALERT_LINK_DASHBOARDS_LABEL_TOOLTIP_CONTENT = i18n.translate(
+  'responseOpsRuleForm.ruleForm.ruleDefinition.alertLinkDashboards.label.tooltipContent',
+  {
+    defaultMessage:
+      'Choose dashboards to link to this rule. Alerts created by this rule will include links to these dashboards.',
```

Comment:
Instead of describing the interaction and result, I think it'd be more useful to explain _why_ we recommend linking to related dashboards. For example:

```suggestion
      'Dashboards let you to visualize and analyze alerts created by this rule. When alerts are created, you can quickly access the dashboards from the alerts' details page. ',
```

Feel free to suggest something different if this ^ is not the direction that you want to go in.

Reply:
@nastasha-solomon I like the general direction of explaining _why_ we recommend linking to related dashboards.

> Dashboards let you visualize and analyze alerts created by this rule

I wanted to mention that when I first read above line, it seemed there will be a dashboard where I can visualize the actual alerts created by this rule. Should we change the wording a bit? For example, Dashboards let you visualize and analyze your data related to this rule to help you investigate an alert?

Reply:
Good point - the first sentence could be interpreted that way. Here's my attempt at a revision:

Investigate and visualize alert data using dashboards. Alerts created by this rule will link to the dashboards you choose.

Reply:
I think that works great! Wdyt @justinkambic?

 Thanks for quickly iterating over it @nastasha-solomon

Reply:
I am ok with this; thanks for sharing your insight and helping us get the context right for this copy @benakansara.